mitmproxy/addons/dumper.py

```diff
@@ -73,6 +73,7 @@ def load(self, loader):
         loader.add_option(
             "dumper_filter", Optional[str], None, "Limit which flows are dumped."
         )
+        ctx.options.content_view_lines_cutoff = 70
```

Comment:
I'd be strongly in favor of re-using the same default across all views. Can we just use the existing default? :)

Reply:
The reason I've made this change is because I felt 512 is a bit too much for mitmdumps's output and better suites for mitmproxy and mitmweb since they have separate consoles to view the responses. For consistency's sake I think I should revert this change

Reply:
Also, this change would break existing tests which expect cutoff by 70 lines